org.eclipse.jdt.junit/src/org/eclipse/jdt/junit/wizards/NewTestSuiteWizardPage.java

```diff
@@ -294,7 +309,8 @@ protected void createClassesInSuiteControl(Composite parent, int nColumns) {
 			gd.horizontalSpan= nColumns-1;
 
 			fClassesInSuiteTable.getTable().setLayoutData(gd);
-			fClassesInSuiteTable.setContentProvider(new SuiteClassesContentProvider(isJUnit4()));
+			System.out.println("is junit4 " + isJUnit4() + " is junit5 " + isJUnit5()); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Please remove the sysout statement.

Reply:
The default radio button selection in the wizard should be New JUnit 5 suite whenever it's possible to create a JUnit 5 suite in the workspace, or the selection in Package Explorer, etc. You can refer to the behavior of New Test Case wizard.

For example, if the selected project contains junit suite bundle then New JUnit 5 suite should be selected by default. 
And when the wizard is invoked on a package or test class(es), if the selection contains a JUnit 5 test, then the wizard should select JUnit 5 suite.

Also, noticed that while editing something in the wizard (like selecting/deselecting test classes, changing suite name, etc.), the radio button selection is getting reset to 4 if 5 was selected before editing. I don't have exact steps to reproduce but it happened a couple of times.

Reply:
@noopur2507 Done and done.  I also fixed the wizard to not always select AllTests which causes an error or warning when the file already exists.  It will default to pick AllTestsxxxx where xxxx is a number that isn't already used.  The user has the option to remove xxxx and overwrite the existing file if desired  I found this annoying while testing and figure a New TestSuite wizard should default to creating a new testsuite.

Reply:
Thanks, Jeff. I tried out the latest patch and it looks good now. We can release it for M3.